slamd/discovery/processing/algorithms/experiment_postprocessor.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import pandas as pd
+
+from slamd.discovery.processing.algorithms.plot_generator import PlotGenerator
+
+
+class ExperimentPostprocessor:
+    @classmethod
```

Comment:
there should be some space (one line) between the class declaration and the method (decorator).

Reply:
Added whitespace